### ISSUE TITLE
Errors even on a fresh project

### ISSUE THREAD

**Issue by lmande** [bakkeby/flexipatch-finalizer#11]
Hey, I had errors using finalizer for dwm-flexipatch. Since this was my first time using finalizer, I didn't know what it was suppose to do. It gave me errors and didn't give me a clean output. Everywhere was full of if statements. Finally I modified it to work but didn't want to push a pr because I can't test if it breaks on a supposed-to-work environment. so here is my tiny debug adventure:

got lots of :
`rm: cannot remove '/home/atom/suckless/o/dwm/patch/layout_monocle.c"': No such file or directory`
Notice the " at the end and saw that it was coming from:
 `schedule_delete(dir substr(f[2], 2, length(f[2]) - 2))`
so I trimmed f[2] at the begining of the loop, after `split(line,f)`
and that fixed it and got myself a clean output of dwm dir.

But i still couldn't "make". it complained about patches.h. I saw your 'sed' at the end but it wasn't working. So this is what I ve changed the sed command to:
`sed -r -i -e 's/ patches.h\s*$//' -e '/^patches.h:\s*$/{N;N;d;}' "${DIRECTORY}/Makefile"`
Although I couldn't see any space after patches.h, patches.h`\s*` s are what fixed it.

I hope this info could be helpful to you

**Comment by bakkeby** [comment]
It might help if you were to show how you used the tool in the first place?

E.g. here is a basic example using fresh clones.

```sh
$ git clone git@github.com:bakkeby/dwm-flexipatch.git
Cloning into 'dwm-flexipatch'...
remote: Enumerating objects: 3720, done.
remote: Counting objects: 100% (1835/1835), done.
remote: Compressing objects: 100% (98/98), done.
remote: Total 3720 (delta 1792), reused 1750 (delta 1737), pack-reused 1885
Receiving objects: 100% (3720/3720), 898.87 KiB | 395.00 KiB/s, done.
Resolving deltas: 100% (2756/2756), done.

$ git clone git@github.com:bakkeby/flexipatch-finalizer.git
Cloning into 'flexipatch-finalizer'...
remote: Enumerating objects: 46, done.
remote: Total 46 (delta 0), reused 0 (delta 0), pack-reused 46
Receiving objects: 100% (46/46), 9.30 KiB | 9.30 MiB/s, done.
Resolving deltas: 100% (27/27), done.

$ cp dwm-flexipatch/patches.def.h dwm-flexipatch/patches.h
$ ./flexipatch-finalizer/flexipatch-finalizer.sh -d dwm-flexipatch -o dwm-finalized --run
$ cd dwm-finalized

$ make
dwm build options:
CFLAGS   = -std=c99 -pedantic -Wall -Wno-unused-function -Wno-deprecated-declarations -Os -I/usr/X11R6/include -I/usr/include/freetype2   -D_DEFAULT_SOURCE -D_BSD_SOURCE -D_POSIX_C_SOURCE=200809L -DVERSION="6.3" -DXINERAMA
LDFLAGS  = -L/usr/X11R6/lib -lX11 -lXinerama -lfontconfig -lXft
CC       = cc
cp config.def.h config.h
cc -c -std=c99 -pedantic -Wall -Wno-unused-function -Wno-deprecated-declarations -Os -I/usr/X11R6/include -I/usr/include/freetype2   -D_DEFAULT_SOURCE -D_BSD_SOURCE -D_POSIX_C_SOURCE=200809L -DVERSION=\"6.3\" -DXINERAMA drw.c
cc -c -std=c99 -pedantic -Wall -Wno-unused-function -Wno-deprecated-declarations -Os -I/usr/X11R6/include -I/usr/include/freetype2   -D_DEFAULT_SOURCE -D_BSD_SOURCE -D_POSIX_C_SOURCE=200809L -DVERSION=\"6.3\" -DXINERAMA dwm.c
cc -c -std=c99 -pedantic -Wall -Wno-unused-function -Wno-deprecated-declarations -Os -I/usr/X11R6/include -I/usr/include/freetype2   -D_DEFAULT_SOURCE -D_BSD_SOURCE -D_POSIX_C_SOURCE=200809L -DVERSION=\"6.3\" -DXINERAMA util.c
cc -o dwm drw.o dwm.o util.o -L/usr/X11R6/lib -lX11 -lXinerama -lfontconfig -lXft
```

**Comment by lmande** [comment]
Hey I think I found the cause. First, I did exactly what you did: 
  

```
$ git clone https://github.com/bakkeby/flexipatch-finalizer.git ff
Cloning into 'ff'...
remote: Enumerating objects: 46, done.
remote: Total 46 (delta 0), reused 0 (delta 0), pack-reused 46
Unpacking objects: 100% (46/46), done.

$ git clone https://github.com/bakkeby/dwm-flexipatch.git dwmx
Cloning into 'dwmx'...
remote: Enumerating objects: 3720, done.
remote: Counting objects: 100% (1460/1460), done.
remote: Compressing objects: 100% (107/107), done.
remote: Total 3720 (delta 1418), reused 1366 (delta 1353), pack-reused 2260
Receiving objects: 100% (3720/3720), 810.81 KiB | 4.05 MiB/s, done.
Resolving deltas: 100% (2762/2762), done.

$ cd dwmx

$ make 
dwm build options:
CFLAGS   = -std=c99 -pedantic -Wall -Wno-unused-function -Wno-deprecated-declarations -Os -I/usr/X11R6/include -I/usr/include/freetype2   -D_DEFAULT_SOURCE -D_BSD_SOURCE -D_POSIX_C_SOURCE=200809L -DVERSION="6.3" -DXINERAMA
LDFLAGS  = -L/usr/X11R6/lib -lX11 -lXinerama -lfontconfig -lXft         
CC       = cc
cp config.def.h config.h
cp patches.def.h patches.h
cc -c -std=c99 -pedantic -Wall -Wno-unused-function -Wno-deprecated-declarations -Os -I/usr/X11R6/include -I/usr/include/freetype2   -D_DEFAULT_SOURCE -D_BSD_SOURCE -D_POSIX_C_SOURCE=200809L -DVERSION=\"6.3\" -DXINERAMA drw.c
cc -c -std=c99 -pedantic -Wall -Wno-unused-function -Wno-deprecated-declarations -Os -I/usr/X11R6/include -I/usr/include/freetype2   -D_DEFAULT_SOURCE -D_BSD_SOURCE -D_POSIX_C_SOURCE=200809L -DVERSION=\"6.3\" -DXINERAMA dwm.c
cc -c -std=c99 -pedantic -Wall -Wno-unused-function -Wno-deprecated-declarations -Os -I/usr/X11R6/include -I/usr/include/freetype2   -D_DEFAULT_SOURCE -D_BSD_SOURCE -D_POSIX_C_SOURCE=200809L -DVERSION=\"6.3\" -DXINERAMA util.c
cc -o dwm drw.o dwm.o util.o -L/usr/X11R6/lib -lX11 -lXinerama -lfontconfig -lXft         

$ cd ..

$ ./ff/flexipatch-finalizer.sh -r -d dwmx -o finaldwm
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_dwmblocks.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_ewmhtags.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/combo.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_holdbar.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_ltsymbol.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_powerline_status.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_powerline_tags.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_status.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_status2d.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_statusbutton.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_statuscmd.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_statuscolors.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_winicon.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_tabgroups.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_tags.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_taglabels.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_taggrid.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_wintitle.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_fancybar.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_flexwintitle.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_wintitle_floating.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_wintitle_hidden.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_awesomebar.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_systray.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_vtcolors.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_wintitleactions.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_layoutmenu.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/aspectresize.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/attachx.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/autostart.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/cfacts.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/cmdcustomize.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/cool_autostart.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/cyclelayouts.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/decorationhints.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/distributetags.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/dragcfact.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/dragmfact.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/dwmc.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/fsignal.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/exresize.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/fakefullscreenclient.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/floatpos.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/focusdir.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/focusadjacenttag.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/focusmaster.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/focusurgent.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/fullscreen.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/inplacerotate.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/ipc.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/ipc/ipc.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/ipc/util.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/insets.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/keymodes.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/killunsel.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/maximize.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/mpdcontrol.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/moveplace.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/moveresize.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/movestack.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/nomodbuttons.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/pertag.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/placemouse.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/push_no_master.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/push.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/reorganizetags.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/restartsig.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/riodraw.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/rotatestack.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/roundedcorners.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/scratchpad.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/scratchpad_alt_1.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/selfrestart.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/setborderpx.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/shiftview.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/shiftviewclients.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/sizehints_ruled.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/sortscreens.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/stacker.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/sticky.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/swallow.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/swapfocus.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/swaptags.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/switchcol.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/tab.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/tagall.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/tagallmon.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/tagothermonitor.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/tagswapmon.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/tapresize.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/togglefullscreen.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/transfer.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/transferall.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/unfloatvisible.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/vanitygaps.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/warp.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/winview.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/zoomswap.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/xkb.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/xrdb.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_bstack.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_bstackhoriz.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_centeredmaster.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_centeredfloatingmaster.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_columns.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_deck.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_fibonacci.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_flextile-deluxe.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_gapplessgrid.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_grid.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_horizgrid.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_monocle.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_nrowgrid.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_tile.h"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_dwmblocks.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_ewmhtags.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/combo.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_ltsymbol.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_powerline_status.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_powerline_tags.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_status.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_status2d.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_statusbutton.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_statuscmd.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_statuscolors.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_winicon.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_tabgroups.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_tags.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_taglabels.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_taggrid.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_wintitle.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_fancybar.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_flexwintitle.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_wintitle_floating.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_wintitle_hidden.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_awesomebar.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_systray.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_holdbar.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_vtcolors.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_wintitleactions.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/bar_layoutmenu.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/aspectresize.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/attachx.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/autostart.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/cfacts.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/cmdcustomize.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/cool_autostart.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/cyclelayouts.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/decorationhints.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/distributetags.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/dragcfact.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/dwmc.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/fsignal.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/exresize.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/fakefullscreenclient.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/floatpos.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/focusadjacenttag.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/focusdir.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/focusmaster.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/focusurgent.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/fullscreen.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/inplacerotate.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/ipc.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/ipc/IPCClient.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/ipc/yajl_dumps.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/ipc/ipc.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/ipc/util.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/insets.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/keymodes.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/killunsel.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/maximize.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/mpdcontrol.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/moveplace.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/moveresize.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/movestack.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/nomodbuttons.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/pertag.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/placemouse.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/push_no_master.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/push.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/reorganizetags.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/restartsig.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/riodraw.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/rotatestack.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/roundedcorners.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/scratchpad.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/scratchpad_alt_1.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/selfrestart.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/setborderpx.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/shiftview.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/shiftviewclients.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/sizehints_ruled.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/sortscreens.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/stacker.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/sticky.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/swallow.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/swapfocus.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/swaptags.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/switchcol.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/tab.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/tagall.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/tagallmon.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/tagothermonitor.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/tagswapmon.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/tapresize.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/togglefullscreen.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/transfer.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/transferall.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/unfloatvisible.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/vanitygaps.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/warp.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/winview.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/zoomswap.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/xkb.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/xrdb.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/dragmfact.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_facts.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_bstack.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_bstackhoriz.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_centeredmaster.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_centeredfloatingmaster.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_columns.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_deck.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_fibonacci.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_flextile-deluxe.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_gapplessgrid.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_grid.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_horizgrid.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_monocle.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_nrowgrid.c"': No such file or directory
rm: cannot remove '/home/atom/flexi/finaldwm/patch/layout_tile.c"': No such file or directory
```

Posted to show what I got at start. But I think the problem is solved because it was about line endings =/  I saw `^M` at the end of each line which probably caused by git. I didn't take my time to setup my config on this computer yet so thats what happened I think.

So after `$ git config --global core.autocrlf false` , I did everything again and all worked like a charm. Pff seemslike I just found a solution for my messed up dir. Hope this would save someones time then

**Comment by bakkeby** [comment]
That is weird. Are you on a Windows system and/or have you ever set that config to true in the past?

Looking at https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration the core.autocrlf setting automatically converts line feeds to `\r\n` in the files checked out. Can't say that I have ever come across this before.

**Comment by lmande** [comment]
I thought I just installed and deserted this one a bit time ago afterwards. But now Ive checked the git config --list and see there is 1 `cola.spellcheck=false` entry there. So maybe I did via git-cola, I don't know =/ Actually this would go faster If I took my time to think. I first got a bash not found error for finalizer script but I fixed it with sed -i so I thought I was good to go because I was already using flexipatch-dwm but more I talk I see what happened there lol. Sorry and thank you for sparing your time for me
